test/bases.py

```diff
@@ -292,23 +292,26 @@ def test_all_state_can_reach_everything(self):
         """Ensure all state can reach everything and complete the game with the defined options"""
         if not (self.run_default_tests and self.constructed):
             return
-        with self.subTest("Game", game=self.game):
-            excluded = self.multiworld.worlds[self.player].options.exclude_locations.value
-            state = self.multiworld.get_all_state(False)
-            for location in self.multiworld.get_locations():
-                if location.name not in excluded:
-                    with self.subTest("Location should be reached", location=location):
-                        reachable = location.can_reach(state)
-                        self.assertTrue(reachable, f"{location.name} unreachable")
-            with self.subTest("Beatable"):
-                self.multiworld.state = state
-                self.assertBeatable(True)
+        with self.subTest("Game", game=self.game, seed=self.multiworld.seed):
+                excluded = self.multiworld.worlds[self.player].options.exclude_locations.value
```

Comment:
Should this actually be indented an extra time now? If so, why?

Reply:
(it's called I got lazy with removing an earlier try/except).